scintilla.pri

```diff
@@ -18,6 +18,13 @@ scintilla {
     }
   }
 
+  BREW=$$(OPENSCAD_HOMEBREW)
+  !isEmpty(BREW) {
+    include(/usr/local/opt/qscintilla2//data/mkspecs/features/qscintilla2.prf)
```

Comment:
Below there's a comment that "qscintilla2.prf which ships with QScintilla is broken for Mac", but here we include it nevertheless.
The broken aspect appears to be that they try to link with `-lqscintilla2_qt5_debug` in debug mode, but don't ship that library.